templates/epub3/toc.xhtml.ejs

```diff
@@ -11,9 +11,21 @@
 <nav id="toc" epub:type="toc">
     <ol>
         <% content.forEach(function(content, index){ %>
+            <% if(!content.excludeFromToc && content.beforeToc){ %>
+                <li class="table-of-content">
+                    <a href="<%= content.href %>"><%= (content.title || "Chapter "+ (1+index)) %><% if(content.author.length){ %> - <small class="toc-author"><%= content.author.join(",") %></small><% }%><% if(content.url){ %><span class="toc-link"><%= content.url %></span><% }%></a>
+                </li>
+            <% } %>
+        <% }) %>
         <li class="table-of-content">
-            <a href="<%= content.href %>"><%= (content.title || "Chapter "+ (1+index)) %><% if(content.author.length){ %> - <small class="toc-author"><%= content.author.join(",") %></small><% }%><% if(content.url){ %><span class="toc-link"><%= content.url %></span><% }%></a>
+            <a href="toc.xhtml">- <%= tocTitle %> -</a>
```

Comment:
Also a small note @cyrilis, about TOC and pre-TOC content. So I've asked around our book design experts about (1) having the actual TOC entry on the TOC and (2) any reason to have pre-TOC contents listed on TOC.

And for the first one, it's not common to have the TOC entry on the TOC, nor for second one is common to have pre-TOC contents listed on the TOC. [1][2]

And, seems to me, IMO, that the lib should try to follow the best practices as closely as possible, so I think we could aim our templates to have this default behaviour. 🙂

Wdyt?

[1] [Reedsy Blog - Anatomy of a Book: Front Matter, Body and Back Matter](https://blog.reedsy.com/front-matter-back-matter-book)
[2] [Wikipedia - Book Design: Front matter](https://en.wikipedia.org/wiki/Book_design#Front_matter)